analyzers/src/Directory.Build.targets

```diff
@@ -9,9 +9,4 @@
     <BinariesFolderInternal>$(BinariesFolder)internal\</BinariesFolderInternal>
   </PropertyGroup>
 
-  <!-- The condition causes the target to only be executed once during the outer build step.
```

Comment:
The idea with setting a flag here didn't work?

Reply:
What was the exact idea? I remember trying something and it didn't work.

Reply:
Basically add `Condition="$(SomeCustomFlag)!='True'"` (wrong syntax, just illustrative) and then inside the target
`<PropertyGroup><SomeCustomFlag>True</...`.

That will try to execute it just once. I'm not sure if it will work as expected. That should be the smallest change. The need for maintaining this on many different files is scattered => hard to keep track of.